src/protocol/tx_format/constants.md

```diff
@@ -14,3 +14,4 @@
 | `MAX_MESSAGE_DATA_LENGTH`   | `uint16` |                 | Maximum length of message data, in bytes.     |
 | `MAX_STORAGE_SLOTS`         | `uint16` | `255`           | Maximum number of initial storage slots.      |
 | `MAX_WITNESSES`             | `uint64` | `16`            | Maximum number of witnesses.                  |
+| `CHAIN_ID`                  | `uint64` |                 | A unique per-chain identifier.                |
```

Comment:
Do we want to calculate `CHAIN_ID` based on the genesis config, or it is a manually typed number?

Reply:
I think it would make the most sense as a genesis config parameter.